src/Features/Core/Portable/DocumentHighlighting/AbstractDocumentHighlightsService.cs

```diff
@@ -157,13 +159,20 @@ private static bool ShouldConsiderSymbol(ISymbol symbol)
 
             var additionalReferences = new List<Location>();
 
-            foreach (var document in documentsToSearch)
+            foreach (var additionalDocument in documentsToSearch)
             {
-                additionalReferences.AddRange(await GetAdditionalReferencesAsync(document, symbol, cancellationToken).ConfigureAwait(false));
+                // 'documentsToSearch' may contain documents from languages other than our own
+                // (for example cshtml files when we're searching the cs document).  Since we're
```

Comment:
I'm not quite following the scenario here. What document languages were involved here?